### DISCUSSION TITLE
losing reading status after upgrading

### DISCUSSION THREAD

**Discussion by frankli998** [Difegue/LANraragi#543] [Q&A]
Hi! I use docker to run LANraragi, and use a image named "watchtower" to upgrade.
I did a backup before upgrading, which is in json format.

However, when I restored the json backup, reading status was lost. So anything I can do in order to backup reding status?

**Comment by Difegue** [comment]
Database backups don't currently transfer reading progression as they're only really meant to carry metadata.  

If you're upgrading Docker images, you should keep the full database between containers by re-using the same bind mounts every time.

**Reply by frankli998** [reply]
What file stores reading status? I believe I have reused same bind mounts...

**Reply by Difegue** [reply]
Reading progression is stored in the database alongside everything else, so the data should be in `database.rdb` / `appendonly.aof`.
